### PR TITLE
Add check-docker-version workflow

### PR DESCRIPTION
**Pull request by skeet70**
Looks like every patch is just setting the DOCKERVAR input, so this should be good.

Tested [here](https://github.com/IronCoreLabs/docker-postgres-exporter/pull/26), seems to be working fine.

### CODE REVIEW

File: .github/workflows/check-docker-version.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# workflow so a human can fix it. This is meant to run on every commit to a PR.
+# The dockervar should appear near the top of the file in a statement like:
+# ARG FOO_VERSION=1.2.3
+# The actual version should just be a semver version with no -buildnumber and no leading "v" or other characters.
+
+name: Check Docker Version
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+"on":
```
Comment (BobWall23) [inline .github/workflows/check-docker-version.yaml:12]: Craig was removing the quotes from these `on` lines.

**Review by BobWall23 — CHANGES_REQUESTED** [review]

File: .github/workflows/check-docker-version.yaml (hunk 1 above)
Comment (giarc3) [reply .github/workflows/check-docker-version.yaml:12]: It works either way, but the quotes are leftover from when we were patching these files and are weird now

**Review by giarc3 — APPROVED** [review]

File: .github/workflows/check-docker-version.yaml (hunk 1 above)
Comment (cjyar) [reply .github/workflows/check-docker-version.yaml:12]: I agree with Craig; it should be fine to remove the quotes, and it's a bit more readable. I can supply lots of detail about YAML parser bugs that made the quotes necessary when we were patching, but you probably don't want to hear it. tl;dr: YAML kinda sucks.

**Review by cjyar — APPROVED** [review]

File: .github/workflows/check-docker-version.yaml (hunk 1 above)
Comment (BobWall23) [reply .github/workflows/check-docker-version.yaml:12]: Consistency seems beneficial to me.

Comment (BobWall23) [inline .github/workflows/check-docker-version.yaml:12]:
```suggestion
on:
```

**Review by BobWall23 — CHANGES_REQUESTED** [review]

**Review by BobWall23 — APPROVED** [review]

**Pull request merged**